test/dotnet-new.Tests/NuGet.tempaspnetpatch.config

```diff
@@ -6,5 +6,6 @@
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="DotnetCore" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="aspnetcore-dev" value="https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json" />
+    <add key="dotnet-corefxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json" />
```

Comment:
This was a fix for specific ASP.NETcore.App versions in the orchestrated build for release/2.1.3xx that took dependencies on non-published packages: System.IO.Pipelines 0.1.0-preview1-180129-1 and System.Text.Encodings.Web.Utf8 0.1.0-preview1-180129-1 - This change may not be necessary in CLI:master.

Reply:
If this is not necessary, we should remove it in 2.1.3xx and once that branch merges back into master, master will be fixed.

Reply:
It is indeed necessary in release/2.1.3xx at this time - we can merge the addition of the line and then merge the removal as you say.